core/src/banking_stage.rs

```diff
@@ -93,6 +93,8 @@ const MAX_NUM_TRANSACTIONS_PER_BATCH: usize = 128;
 const NUM_VOTE_PROCESSING_THREADS: u32 = 2;
 const MIN_THREADS_BANKING: u32 = 1;
 
+const SLOT_BOUNDARY_CHECK_MS: u64 = 10;
```

Comment:
minor nit: Since [`Duration::from_millis()`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_millis) is `const`, I find it preferable to use a strongly-typed constant vs a plain integer. This will make it safer for future maintainers as well.

I.e.

```rust
const SLOT_BOUNDARY_CHECK_PERIOD: Duration = Duration::from_millis(10);
```
(I'm not stuck on the constant's name, so feel free to tweak that for what feels best.)

Reply:
fixed!